wiki/Beatmap/Song_compilation/ja.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+tags:
+  - song compilations
+---
+
+# Song compilation
+
+**Song compilation**はいろいろな曲を一つにまとめたもので、通常はジャンルやアーティスト、その他のテーマに沿ってまとめられます。
+
+Song compilationのビートマップは、曲をまとめたものを1つのオーディオファイルにしています。通常は、それぞれの曲のコーラスや小節などの一部分を使います。フォルダ内に複数の曲ファイルを含むビートマップと混同しないでください。
```

Comment:
missing link

for ref

`[Beatmaps](/wiki/Beatmap) of song compilations use a single audio file that contains the whole collection of songs, usually including only a single chorus or verse of each. This is not to be confused with a beatmap that includes multiple song files in its folder.`